Cargo.toml

```diff
@@ -28,6 +28,7 @@ uuid = { version = "0.8", features = ["v4", "serde"] }
 url = "2.2.2"
 function_name = "0.2.0"
 thiserror = "1.0.30"
+regex = "1.5.4"
```

Comment:
Aie aie aie

Reply:
<3

Reply:
What's the problem?

Reply:
Not a big fan of regex, comes with an entire ecosystem, if we have only a very small use case and can find another alternative, that's my way to go, but if there is no other solution, then ... but IMO, we should try to avoid it while we can.